.kokoro/tests/run_tests_java11.bat

```diff
@@ -1,4 +0,0 @@
-setx JAVA_HOME "C:\\OpenJDK\jdk-11.0.11"
-setx PATH "%JAVA_HOME%\bin;%PATH%"
-call RefreshEnv.cmd
-"C:\Program Files\Git\bin\bash.exe" github/cloud-sql-jdbc-socket-factory/.kokoro/tests/run_tests.sh
```

Comment:
do these files being removed need to be part of this PR?

Reply:
they can be in the other one, but I thought it made sense to leave these deletions in this PR since the other script in this folder needs to be moved in this PR

Reply:
Sounds good, no worries